app/Models/QueryBuilders/SearchFormQueryBuilder.php

```diff
@@ -112,15 +146,15 @@ public function buildQuery(): array
         $andQueries = [];
         $boolQueries = [];
 
-        if (isset($this->isPublic) && !empty($this->isPublic)) {
+        if (isset($this->isPublic) && is_bool($this->isPublic)) {
```

Comment:
Is there any case when isPublic is set but not boolean type?

Reply:
Removed check for "is_bool".